ci/common/test.sh

```diff
@@ -80,8 +80,8 @@ valgrind_check() {
   check_logs "${1}" "valgrind-*"
 }
 
-asan_check() {
-  if test "${CLANG_SANITIZER}" = "ASAN_UBSAN" ; then
+sanitizer_check() {
```

Comment:
since we're renaming it let's put the verb first, similar to `check_logs`.
```suggestion
check_sanitizer() {
```